public/toolkit/styles/atoms/_tables.scss

```diff
@@ -79,7 +79,7 @@ table {
   }
 
   // Remove zebra coloring and adds border
-  &.td-plain, 
+  &.td-plain,
```

Comment:
@akegan Is this just removing trailing whitespace?

Reply:
yep! my editor is setup to remove all trailing whitespaces, so it just made this update on save. thanks for the review!